### PR TITLE
Update documentation for testing the ARO operator

### PR DESCRIPTION
**Pull request by lranjbar**
### Which issue this PR addresses:

N/A

### What this PR does / why we need it:

This PR updates the documentation around the operator and Fedora 37. I found some issues with the docs while creating a new controller for ARO-3112.

### Test plan for issue:

N/A

### Is there any documentation that needs to be updated for this PR?

No. This is to update the developer docs. 😄

### CODE REVIEW

**Review by SudoBrendan — APPROVED** [review]

**Review by jhoreman — APPROVED** [review]
Small comment but won't hold approval over it:

`/bin/sh` is often symlinked to another shell depending on the flavour of your distro. As long as we don't truly expect `bash` style implementation (and `dash` works just fine, for example) then this looks fine to me!

File: docs/operators.md [hunk 1]
```diff
@@ -66,13 +66,24 @@ Make sure KUBECONFIG is set:
...
-Then do:
+Scale the operator:
 ```sh
 oc scale -n openshift-azure-operator deployment/aro-operator-master --replicas=0
+```
+
+Build the operator binary and run it locally (as if it was running a master node)
+```sh
 make generate
 go run ./cmd/aro operator master
```
Comment (cadenmarchese) [inline docs/operators.md:88]: One more thing I think we can add here that's required for MacOS / M1 users (it isn't required on Linux, but still works just as well):
```suggestion
go run -tags containers_image_openpgp ./cmd/aro operator master
```
We also build ARO with this tag, but it looks like we forgot to update the docs here.

**Review by petrkotas — APPROVED** [review]
Thanks @lranjbar !

**Comment by petrkotas** [comment]
Merging the docs update, ignoring e2e because this PR does not modify any code.

**Pull request merged**